src/ucs/datastruct/bitmap.h

```diff
@@ -254,8 +262,8 @@ static inline bool _ucs_bitmap_assert(bool condition)
  * @return Bit value (0 or 1)
  */
 #define UCS_BITMAP_GET(_bitmap, _bit_index) \
-    (_ucs_bitmap_assert(_bit_index < _UCS_BITMAP_NUM_WORDS(_bitmap) * \
-                                             UCS_BITMAP_BITS_IN_WORD) && \
+    (_ucs_bitmap_assert( \
```

Comment:
wrap macro body with ({

Reply:
It then can't be used inside `EXPECT_EQ(...)` 
(`statement-expressions are not allowed outside functions nor in template-argument lists...`)
and the test code would have to be changed accordingly. Do we prefer that?

Reply:
i see 
maybe convert UCS_BITMAP_WORD_INDEX to inline function which will contain the assertion?